test/config.js

```diff
@@ -0,0 +1,22 @@
+import { EnvironmentType } from "@verida/account"
+
+export default {
```

Comment:
Need a way to set these from the `.env` file.

Reply:
Why? These are configs just for testing which serve a different purpose.

Reply:
Because I had to change the URL of the server to post 5000 (which is possible using the `PORT` config setting) but there was no way to point the tests at it without changing the sourcecode. 

Perhaps there should be `.tests.env` or something separately.

Reply:
Changed to load port from process.env for tests.